.github/workflows/main.yml

```diff
@@ -32,10 +33,8 @@ jobs:
           ($env:REPO_NAME) >> VERSION
           ($env:BRANCH) >> VERSION
           ($env:COMMIT) >> VERSION
-      - name: Build Dalamud
+      - name: Build and Test Dalamud
```

Comment:
Is it not possible to keep these as separate steps?

Reply:
Based on the runner behavior it wasn't, since the test was unable to find the previously built binaries and was triggering a rebuild, causing log duplication when the warnings appeared again. When combined this way, Nuke can handle all the steps at once instead of being called twice. I can look further into it though if needed.

Reply:
This specific one isn't a blocking thing, but I can see why this'd be annoying for local dev work especially as Nuke is somewhat required for certain workflows.

Ideally we'd use them as separate steps though. I'd say see if it's doable but don't worry _too_ much if it's not (unless we change our minds/goat says we need it split).

Reply:
Mentioned on discord, but I couldn't figure out how to make Nuke handle the artifact upload/download portion required for the splitting to work, so I added `--skip-tests` to mimic the compile-only functionality from before, and `build.ps1 test` still works as usual.